packages/react/src/themes/teams-dark/components/Toolbar/toolbarVariables.ts

```diff
@@ -0,0 +1,11 @@
+import { ToolbarVariables } from '../../../teams/components/Toolbar/toolbarVariables'
+import { colors } from '../../../teams/colors'
+
+export default (siteVars: any): Partial<ToolbarVariables> => ({
+  foreground: siteVars.colors.grey[250],
+  dividerBorder: siteVars.colors.grey[450],
+  foregroundHover: siteVars.colors.brand[400],
+  foregroundActive: siteVars.colors.brand[400],
+  foregroundDisabled: colors.grey[450],
```

Comment:
shouldn't we consume all colors from `siteVariables`?

Reply:
Yup, I plan to discuss the usage of color palette with @mnajdova and update the variables accordingly.

Reply:
As this is a new component, we can start with the using the color scheme, and do only the necessary overrides in the different theme, if there is any 👍

Reply:
Switched to color scheme